CMakeLists.txt

```diff
@@ -184,14 +192,15 @@ if (ONEDPL_BACKEND MATCHES "^(tbb|dpcpp|dpcpp_only)$")
             set(USE_FSYCL_OPTION TRUE)
         endif()
 
-        # check device type
+        # check device type for oneDPL test targets
         if (ONEDPL_DEVICE_TYPE MATCHES "^(CPU|GPU|FPGA_EMU|FPGA_HW)$")
```

Comment:
Does it make sense to warn or throw an error if this `ONEDPL_DEVICE_TYPE` is specified at the same time as `_ONEDPL_PSTL_OFFLOAD`?

Reply:
I added a check against mutual set of `_ONEDPL_PSTL_OFFLOAD` and `ONEDPL_DEVICE_TYPE`.

Reply:
Looks good.